tests/suites/test_suite_net.function

```diff
@@ -71,6 +71,23 @@ void context_init_free( int reinit )
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void context_set_fd( int fd )
+{
+    mbedtls_net_context ctx;
+
+    mbedtls_net_init( &ctx );
+
+    /* Test that mbedtls_net_set_fd with a "valid" file descriptor and an
+     * invalid one */
+    TEST_ASSERT( mbedtls_net_set_fd( &ctx, fd ) == 0 );
```

Comment:
since you're testing only two possible inputs for this function (1 and -1) you can either:
* put expected result as function input `context_set_fd( int fd, int result)` and extend `.data` file with
```
context_set_fd:1:0
context_set_fd:-1:MBEDTLS_ERR_NET_INVALID_CONTEXT
```
* or hardcode both files descriptors inside the test case ( since it's quite small ):
```
TEST_ASSERT( mbedtls_net_set_fd( &ctx, 1 ) == 0 );
TEST_ASSERT( mbedtls_net_set_fd( &ctx, -1 ) == MBEDTLS_ERR_NET_INVALID_CONTEXT );
```
and inside `.data`: `context_set_fd:`

up to you which one you pick.

Reply:
Thank you for your suggestion, I'm not used to those test.